book/07-error-handling.html

```diff
@@ -190,7 +190,7 @@ <h2>Error and Or_error</h2>
           <link rel="import" href="code/error-handling/main.topscript" part="7"/>
 
           <p>
-	    Note that the time isn't actually serialized into an
```

Comment:
Good catch, but maybe better, "Note that the character isn't actually..." rather than the more generic "type"?

Reply:
"character" is definitely better, I had the same thoughts shortly after submitting.